litellm/llms/vertex_ai_anthropic.py

```diff
@@ -236,17 +236,19 @@ def completion(
         if client is None:
             if vertex_credentials is not None and isinstance(vertex_credentials, str):
                 import google.oauth2.service_account
-
-                json_obj = json.loads(vertex_credentials)
-
                 creds = (
                     google.oauth2.service_account.Credentials.from_service_account_info(
-                        json_obj,
+                        json.loads(vertex_credentials),
                         scopes=["https://www.googleapis.com/auth/cloud-platform"],
                     )
                 )
                 ### CHECK IF ACCESS
                 access_token = refresh_auth(credentials=creds)
+            else:
+                import google.auth
+                creds, _ = google.auth.default()
+                ### CHECK IF ACCESS
+                access_token = refresh_auth(credentials=creds)
```

Comment:
This PR is failing our testing:https://app.circleci.com/pipelines/github/BerriAI/litellm/8812/workflows/18472db4-fb2b-4c49-83c5-e43db2c447be/jobs/18373/tests#failed-test-0